packages/config/src/loadConfigFromFile.ts

```diff
@@ -76,43 +85,49 @@ export async function loadConfigFromFile({
     resolvedPath =
       ['prisma.config.ts'].map((file) => path.resolve(configRoot, file)).find((file) => fs.existsSync(file)) ?? null
 
-    if (!resolvedPath) {
+    if (resolvedPath === null) {
       debug(`No config file found in the current working directory %s`, configRoot)
 
       return { resolvedPath }
     }
   }
 
-  const { required, error } = await requireTypeScriptFile(resolvedPath)
+  try {
+    const { required, error } = await requireTypeScriptFile(resolvedPath)
 
-  if (error) {
-    return {
-      resolvedPath,
-      error,
+    if (error) {
+      return {
+        resolvedPath,
+        error,
+      }
     }
-  }
 
-  debug(`Config file loaded in %s`, getTime())
+    debug(`Config file loaded in %s`, getTime())
 
-  try {
-    // TODO: We should parse `configExport` to ensure it conforms to `PrismaConfig`'s shape at runtime
-    if (required['default'] === undefined) {
+    // Ensure the config file conforms to the expected PrismaConfig schema.
+    const parseResultEither = parsePrismaConfig(required['default'])
+
+    if (Either.isLeft(parseResultEither)) {
```

Comment:
I think I don't get the concept of how effect treats left vs right here. 🤔 To me their definition is inverted and this should be a `Either.right(...)`. What am I missing?